monai/handlers/clearml_handlers.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from .tensorboard_handlers import TensorBoardHandler, TensorBoardImageHandler, TensorBoardStatsHandler
+
+
+class ClearMLStatsHandler(TensorBoardStatsHandler):
+    """
+    Class to write tensorboard stats into ClearML WebUI.
+    """
+
+    def __init__(self, project_name ="Muhammad/monai", task_name="unet_training_dict", output_uri=True, *args, **kwargs):
```

Comment:
Please change the default project name :) Maybe something like `Monai AI`.
Also give it a general task name.

Reply:
Yeah, the default project name will be changed. The current is one given for draft pull-request. As there is permission issues, in our allegro master.